web/app/routes/job.js

```diff
@@ -37,9 +98,38 @@ export default Ember.Route.extend({
         })
       });
     },
+    submitUserChanges(job) {
+      var jobDefId = job.get("jobdefid");
+      var schedulerName = job.get("scheduler");
+      var cluster = job.get("cluster");
+      const cookieName = "elephant." + cluster + ".session.id"
+      var scheduler = new Scheduler();
+      var schedulerUrl = scheduler.getSchedulerUrl(jobDefId, schedulerName)
+      if (!Cookies.get(cookieName)) {
+        this.doLogin(schedulerUrl, cluster)
+      } else {
+        var userAuthorizationStatus = this
+            .getUserAuthorizationStatus(jobDefId, schedulerUrl, cookieName)
+        if (userAuthorizationStatus === "authorised") {
+          //call the param change function
+        } else if (userAuthorizationStatus === "unauthorised") {
+          alert("User is not authorised to modify TuneIn details!!");
+        } else if (userAuthorizationStatus === "session_expired") {
+          //Removing the existing session_id Cookie
+          Cookies.remove(cookieName)
+          this.doLogin(schedulerUrl, cluster)
+        } else if (userAuthorizationStatus === "error") {
+          alert("Some error occured while trying to Authorization!!")
```

Comment:
nit: avoid use of blocking UI's. Prefer inline error messages.

Reply:
The alerts have been now changed to Notifications.